packages/salesforce/CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # @openfn/language-salesforce
 
+## 6.0.0
+
+### Major Changes
```

Comment:
@mtuchi does the markdown in this changelog look right to you? I think it's gotten a bit jumbled up during the merge (this tends to happen)

Can you take a look at the structure and heading levels and make sure it reads clearly?

Maybe check http for a reference https://docs.openfn.org/adaptors/packages/http-changelog#700

Reply:
Hiya @josephjclark i have update the changelog, please have a look

Reply:
Much better, thank you!